Tests/SwiftCliCoreTests/NestedArrayBuilderTest.swift

```diff
@@ -49,4 +59,13 @@ class MultipleStitchExpanderTest: XCTestCase {
                     invalidStitchNumber: "x")
         ))
     }
+
+    class MultipleStitchExpanderTest: XCTestCase {
+        func testExpandValidMultipleStitch() throws {
+            let result = try NestedArrayBuilder().expandRow(row: ["k4", "p1"])
+            expect(result).to(equal(["k1", "k1", "k1", "k1", "p1"]))
+        }
```

Comment:
Is this a copy-paste mistake?

Reply:
I don't think so... unless I'm missing something?

Reply:
Sorry, I just realized that was unclear. What are you referring to with the copy paste mistake? I think I might be missing what you're talking about

Reply:
It looks like you have `class MultipleStitchExpanderTest` inside a larger `class MultipleStitchExpanderTest` and this specific test itself `testExpandValidMultipleStitch` is duplicated as well.

Reply:
Oh, yes 🤦‍♀️ Sorry, thank you for catching that.